src/spi.rs

```diff
@@ -448,9 +469,19 @@ impl<SPI: Spi, SCLK: OutputPin, SDO: OutputPin, SDI: InputPin + OutputPin, CS: O
         read_len: usize,
         write: *const u8,
         write_len: usize,
+        _keep_cs_active: bool,
     ) -> Result<(), SpiError> {
+        let flags = 0;
+
+        #[cfg(any(esp_idf_version = "4.4", esp_idf_version_major = "5"))]
```

Comment:
Can you turn this check into !V4.3?
I.e. something like `#[cfg(not(esp_idf_version = "4.3"))]`

This way, if this code is still around when ESP-IDF V6 is released, it won't break (and we don't support ESP-IDF releases < 4.3).